parser/src/main/java/me/vertretungsplan/parser/IphisParser.java

```diff
@@ -224,13 +240,28 @@ void parseIphis(SubstitutionSchedule substitutionSchedule, JSONArray changes, JS
                 teachersHashMap.put(teacher.getString("id"), teacher.getString("name"));
             }
         }
+
+        // Add Messages
+        List<AdditionalInfo> infos = new ArrayList<>(messages.length());
+
+        for (int i = 0; i < messages.length(); i++) {
+            JSONObject message = messages.getJSONObject(i);
+            AdditionalInfo info = new AdditionalInfo();
+            info.setHasInformation(message.getBoolean("notification"));
+            info.setTitle(message.getString("titel").trim());
+            info.setText(message.getString("nachricht").trim());
+            infos.add(info);
+        }
+
+        substitutionSchedule.getAdditionalInfos().addAll(infos);
+
         // Add changes to SubstitutionSchedule
         LocalDate currentDate = LocalDate.now();
         SubstitutionScheduleDay substitutionScheduleDay = new SubstitutionScheduleDay();
         substitutionScheduleDay.setDate(currentDate);
+        substitutionScheduleDay.setLastChange(LocalDateTime.now());
```

Comment:
Der lastChange ist eigentlich dazu gedacht, anzuzeigen, wann das letzte Mal tatsächlich von der Schule etwas am Plan geändert wurde (wird in der App als "Stand: ..." angezeigt). Liefert die API diese Info auch?
Der lastChange kann wahlweise auch für den gesamten Plan statt für die einzelnen Tage gesetzt werden. Wenn er für den ganzen Plan gesetzt wurde, wird der angezeigt, wenn er für die einzelnen Tage gesetzt wird dann das neueste Datum davon.